src/Facets/AmarokFacet.sol

```diff
@@ -42,10 +37,8 @@ contract AmarokFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
 
     /// @notice Initialize the contract.
     /// @param _connextHandler The contract address of the connext handler on the source chain.
-    /// @param _srcChainDomain The domain of source chain.
-    constructor(IConnextHandler _connextHandler, uint32 _srcChainDomain) {
+    constructor(IConnextHandler _connextHandler) {
         connextHandler = _connextHandler;
-        srcChainDomain = _srcChainDomain;
```

Comment:
if you remove this (which is correct) then you also need to adjust the deploy script

Reply:
you are right.
I updated the deployment script of Amarok facet.
https://github.com/lifinance/contracts/pull/214/commits/31a195a828391171d0fb240aa0a39f700d6256df